### ISSUE TITLE
Add screen reader text to default variable price option

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/easy-digital-downloads#4340]
There are couple of issues when you enable variable pricing on download edit screen. 

1. Default variable price can't be set or focused using keyboard only. Test can be done by using only `Tab`, `Shift+Tab`, `Enter`, `Space` or arrow keys. I need to look into this more closely. There is `<input type="radio">` but there is no label for it. I was thinking something like this but even this doesn't enable focus on default variable price.

```
<label class="selectit">
		<input type="radio" <?php checked( $default_price_id, $key, true ); ?> class="edd_repeatable_default_input" name="_edd_default_price_id" value="<?php echo $key; ?>" />
		<span class="screen-reader-text"><?php printf( __( 'Set ID %s as default price', 'easy-digital-downloads' ), $key ); ?></span>
</label>
```

2. You can't "Drag and Drop" variable prices in different order using keyboard only. This could be hard to solve. I look into it.

**Comment by samikeijonen** [comment]
Sorry, you **can** set default variable price using keyboard only.

![variable_pricing](https://cloud.githubusercontent.com/assets/1820415/13252537/01d19efa-da40-11e5-80f9-e25371d90a6c.png)

I was just confused that focus stopped first time in the second variable price (checked one). From that point you can use up/down arrow keys to select another price option. But I would still add label and `screen-reader-text`. Otherwise screen reader users have no idea what that radio button does.
